web/themes/joinup/templates/search_api/facets-type-wrapper--more.html.twig

```diff
@@ -12,8 +12,13 @@
  * @ingroup themeable
  */
 #}
+{% if items|length > 2 %}
+  {% set class='tab__container tab__container--wide' %}
```

Comment:
First, apologies for not seeing your question on the jira ticket earlier. Please adapt the spacing of the tabs to be as the design, which is new after the changes in the grid.